packages/features/form-builder/fieldsThatSupportLabelAsSafeHtml.ts

```diff
@@ -0,0 +1,6 @@
+import type { FieldType } from "./schema";
+
+/**
+ * Once a component supports labelAsSafeHtml, add it's field's type here.
+ */
+export const fieldsThatSupportLabelAsSafeHtml: FieldType[] = ["boolean"];
```

Comment:
Doesn't want to support in all components right now as all those Components will need to use dangerouslySetInnerHTML then. Would prefer to use it only where needed.